examples/sync-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@automerge/example-sync--server",
```

Comment:
is the double `--` deliberate here? Cosmetic, I know

Reply:
Good spot